examples/linear_model/plot_bayesianridge_curvefit.py

```diff
@@ -0,0 +1,82 @@
+"""
+============================================
+Curve Fitting with Bayesian Ridge Regression
+============================================
+
+Computes a Bayesian Ridge Regression of Sinusoids.
+
+See :ref:`bayesian_ridge_regression` for more information on the regressor.
+
+In general, when fitting a curve with a polynomial by Bayesian ridge
```

Comment:
To explain the role of alpha_init and lambda_init, I would  add something like: The computation of the regularization parameters (alpha, lambda) is an iterative procedure that depends on initial values.